Makefile.am

```diff
@@ -740,10 +741,13 @@ EXTRA_DIST += \
 	man/glib-event-glue.c
 
 # ------------------------------------------------------------------------------
+rootlibexec_LTLIBRARIES += \
+	libshared.la
```

Comment:
hmm, we stick this in rootlibexec, i.e. /usr/lib/systemd. Is ldconfig OK with that, how does that work?